admin/tools/docker/deployment/in2p3/runQueries.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2015 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+"""
+A test program that runs queries in parallel. The queries will run
+with the database we have on the IN2P3 cluster for the Summer 2015
+test.
+
+@author  Jacek Becla, SLAC
+"""
+
+# -------------------------------
+#  Imports of standard modules --
+# -------------------------------
+import argparse
+import logging
+import os
+import pprint
+import random
+import threading
+import time
+
+# ----------------------------
+# Imports for other modules --
+# ----------------------------
+import MySQLdb
```

Comment:
This is only a test script on the client side, will client have to use db? If yes, I'll create a ticket for that.